### ISSUE TITLE
Custom keymappings don't remove the default keymapping

### ISSUE THREAD

**Issue by JosephTLyons** [zed-industries/community#129]
If you customize an action's key combo, the old key combo still exists. This doesn't cause any actual issues, but it leaves more stuff around that can wind up getting triggered by accident.

**Comment by JosephTLyons** [comment]
In v0.94.0, key bindings can be removed via `null.`

**Comment by liebkne** [comment]
@JosephTLyons Does not seem to work for me (0.94.4).

```
{
    "context": "Editor && mode == full",
    "bindings": {
      "ctrl-g": null
    }
}
```
still invokes `go_to_line::Toggle`

and 

```
{
    "bindings": {
      "cmd-,": null
    }
}
```

still opens settings.

**Comment by SomeoneToIgnore** [comment]
Thank you, I've looked at the issue and it seems that Mac menu items (top toolbar) live a bit with their own life: anything there with the default shortcut will have them for a while.
I have some ugly fix that proved the hypothesis, so hopefully we'll release a proper fix next week.

**Comment by SomeoneToIgnore** [comment]
We have a fix coming with Zed 0.94, hopefully today.
All three keybindings from https://github.com/zed-industries/community/issues/129#issuecomment-1636591223 are suppressed now, and I don't see any other issues related.

Thank you again for the report.

**Comment by liebkne** [comment]
@SomeoneToIgnore Thanks for you work! I can finally rebind those keys!
